language core;

extend with #exceptions, #exception-type-declaration;

exception type = Bool

fn main(n : Nat) -> Bool {
  return throw(succ(0))
}

